Makefile

```diff
@@ -96,6 +96,8 @@ ifdef HOST_WINDOWS
   #
   # TODO do not do this, and instead do fine-grained export annotations.
   GLOBAL_LDFLAGS += -Wl,--export-all-symbols
+  # Increase the default reserved stack size to 128 MB so Nix doesn't run out of space
+  GLOBAL_LDFLAGS += -Wl,--stack,134217728
```

Comment:
Can we do this per executable? So change `src/nix/local.mk`? The one with the unit tests can be adjusted too.

Also maybe do the trick in https://stackoverflow.com/questions/1926063/how-do-i-perform-arithmetic-in-a-makefile to make things normal?

Reply:
> Can we do this per executable? So change `src/nix/local.mk`? The one with the unit tests can be adjusted too.

Done, I added the flag to nix and all the `libnix*-tests` binaries. However, I'm not sure if that's what you're referring to. Do you mean all the tests or just the `libnixutil-tests` binary?